### PR TITLE
Refine POSTBUILD for spresense and esp32-core

### PR DESCRIPTION
**Pull request by xiaoxiang781216**

### CODE REVIEW

File: boards/xtensa/esp32/esp32-core/Kconfig [hunk 1]
```diff
@@ -38,4 +38,10 @@ config ESP32CORE_RUN_IRAM
 		allows interoperability with the esp-idf system but makes you
 		reliant on the esp-idf design for these parts. Both are possible.
 
+config ESP32CORE_BINARY
+	bool "esp32 binary format"
```
Comment (yamt) [inline boards/xtensa/esp32/esp32-core/Kconfig]: this is misleading as it's just a flash image.
how about "Flash image (mainly for QEMU)"

Comment (xiaoxiang781216) [reply boards/xtensa/esp32/esp32-core/Kconfig]: Done.

**Comment by jerpelea** [comment]
+1 but it failed the checks

**Comment by Ouss4** [comment]
>  Will the build tests restart automatically so that we can confirm that?

I re-run the CI checks.

**Pull request merged**